### ISSUE TITLE
Add Note for Rust RFC

### ISSUE THREAD

**Issue by hasezoey** [magiclen/path-absolutize#7]
I think this crate (along with [`path-dedot`](https://crates.io/crates/path-dedot)) is similar what is proposed as a RFC in rust itself in https://github.com/rust-lang/rfcs/issues/2208

If it is actually the case that this crate (`path-absolutize`) is similar (or exactly) to the RFC, it maybe should be mentioned in the README / docs

**Comment by magiclen** [comment]
Yeah, they're related. This crate does the same kind of thing the RFC talks about: make a path absolute and clean up `.` / `..` without touching the filesystem.

One update since you opened this: `std` now has `std::path::absolute` (stable since Rust 1.79). But it's not the same thing. It doesn't fully remove dots the way this crate does — for example, on Unix it leaves `..` in the path. `path-absolutize` still resolves those. So `std` doesn't cover this crate's job yet.
